packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2817,6 +2818,14 @@ export class ContainerRuntime
 				forcedFullTree,
 			} as const;
 
+			// Validate that the summary generated by the summarizer nodes is correct.
+			const validateResult = this.summarizerNode.validateSummary();
+			if (!validateResult.success) {
+				const { success, ...loggingProps } = validateResult;
+				const error = new LoggingError(validateResult.reason, { ...loggingProps });
```

Comment:
It's a little creepy (not typesafe, and dependent on implementation of `LoggingError`'s handling of telemetry props) to plumb `retryAfterSeconds` through via `loggingProps`.  We have `ThrottlingError` which has strongly-typed `retryAfterSeconds`, and also slaps `canRetry: true` on the error which definitely applies (but apparently isn't necessary).

I don't have a great tactical fix in mind, unfortunately.  The problem is basically that the retry props on errors are implicit.  They're explicit on the Driver errors, but those all have `errorType`s which don't apply (and it's ok for this error to not have `errorType` I think, at least for now).

Well, this could work? At least it's explicit that these properties matter.

```suggestion
				const { success, retryAfterSeconds, ...loggingProps } = validateResult;
				const retryProps = retryAfterSeconds === undefined ? undefined : { retryAfterSeconds, canRetry: true };
				const error = Object.assign(new LoggingError(validateResult.reason, loggingProps), retryProps);
```

Reply:
Added a new error type "RetriableSummaryError" to summaryGenerator.ts that does this a bit more cleanly. Please take a look and let me know if its too much to create a separate error type for this.
The idea is that other errors during summary can use this and indicate whether the error can be retried.

Reply:
Great idea!